### DISCUSSION TITLE
Is there a binary download for Microcks?

### DISCUSSION THREAD

**Discussion by hardikgw** [microcks/microcks#894] [Q&A] [answered]
I am installing microcks on existing kafka, mongo, keycloak. Where do I get binary files for microcks? Do I have to build it from source?

**Comment by lbroudoux** [comment, marked as answer]
Hey there!
Microcks is distributed as container images only. We provide different means for installing it depending if you're targeting your laptop or a server. Basically on a laptop, the easiest way to start with is via Docker Compose that is documented here: https://microcks.io/documentation/installing/docker-compose/

Hope it helps!

**Reply by cvgaviao** [reply]
Hey @lbroudoux , where can I find the image for version [1.8.0] ?

On dockerhub the latest one is 1.4.1 (https://hub.docker.com/r/microcks/microcks/tags)  :(

**Reply by yada** [reply]
Hi @cvgaviao,
All microcks images are on quay.io, see: https://quay.io/repository/microcks/microcks?tab=tags&tag=latest

Docker hub is not updated and in sync clearly... we should maybe sync both repositories, can you create an issue for this purpose so we can see if the community need it or not?
Thanks 
Yacine
